lib/graphql/client.ex

```diff
@@ -29,7 +30,8 @@ defmodule BridgeEx.Graphql.Client do
   @spec call(
           url :: String.t(),
           query :: String.t(),
-          variables :: map() | String.t(),
+          variables :: map(),
+          encode_variables :: boolean(),
```

Comment:
So you chose the solution with another argument... 🙂

Only two things:
1. this is a breaking change, as this module could be directly used by someone, so we should increase the major.
2. since the breaking change, why not merging all the options in a single keyword list? for instance:
```
spec call(
          url :: String.t(),
          query :: String.t(),
          variables :: map(),
          opts :: Keyword.t()
	)

call(url, query, vars, encode_variables: true, http_headers: %{"A": "a"}, http_options: ..., retry_options: ....) 
```

Reply:
Fortunately [it doesn't seem to be used directly yet](https://cs.github.com/?scope=org%3Aprimait&scopeName=primait&q=BridgeEx.Graphql.Client), but it's a breaking change nevertheless and you are right that we could go all-in and add an `opts` parameter